### DISCUSSION TITLE
[Accessibility-Meta] How plans for accessibility improvements will be translated?

### DISCUSSION THREAD

**Discussion by abhina-kumar** [w3c/personalization-semantics#256] [Ideas]
Accessibility-Metadata states that it will have details about "Plans for further accessibility improvements". Will this be translated?

**Comment by lwolberg** [comment]
We don't know yet. 

In today's practice, plans for accessibility improvement are often included in the accessibility statement. 
We can consider adding this as a separate page.

**Comment by lwolberg** [comment]
To be taken up after Well Known Destination is, at least, FPWD
